docs/source/design/python_imports.rst

```diff
@@ -0,0 +1,61 @@
+.. -*- mode: rst -*-
+.. vi: set ft=rst sts=4 ts=4 sw=4 et tw=79:
+
+.. _chap_design_python_imports:
+
+************************
+Python import statements
+************************
+
+.. topic:: Specification scope and status
+
+   This specification describes the current (albeit incomplete) implementation.
+
+The following rules apply to any ``import`` statement in the code base:
+
+- All imports *must* be absolute.
+
+- Imports *must* be placed at the top of a source file, unless there is a
+  specific reason not to do so (e.g., delayed import due to performance
+  concerns, circular dependencies). If such a reason exists, it *must*
+  should be documented by a comment at the import statement.
+
+- There *must* be no more than one import per line.
+
+- Multiple individual imports from a single module *must* follow the pattern::
+
+      from <module> import (
+          symbol1,
+          symbol2,
+      )
+
+  Individual imported symbols *should* be sorted alphabetically. The last symbol
+  line *should* end with a comma.
+
+- Imports from packages and modules *may* be grouped in categories like
```

Comment:
I think, while `may be grouped` is proper description of status quo, it should actually be `should be grouped` at least.
Not only does PEP8 that as well, but more importantly: There's little to no utility in doing it sometimes and sometimes not. Either it's something that one can rely on to get a quick grasp or I need to inspect every file's import closely to know.

Reply:
```suggestion
- Imports from packages and modules *should* be grouped in categories like
```